jaseci_core/jaseci/utils/jsorc.py

```diff
@@ -0,0 +1,180 @@
+from .promon import Promon
+import time
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+import os
+import yaml
+import multiprocessing
+from .utils import logger
+
+
+class KubeController:
```

Comment:
@Gorgeous-Patrick Can you add some documentation for the modules and functions in this file and promon? Nothing too complicated, just 1 or 2 sentences as docstring """ for each function.

Reply:
Sure I will do that..